modules/govuk/manifests/node/s_gatling.pp

```diff
@@ -0,0 +1,73 @@
+# == Class: govuk::node::s_gatling
+#
+# This machine class is used to run Gatling, a load testing tool.
+#
+# === Parameters
+#
+# [*root_dir*]
+#  The root of the website where Gatling stores its results
+#  Default = '/home/gatling/gatling-software/results'
+#
+# [*repo*]
+#  The github repository where GOV.UK defines its load tests
+#  Default = 'git@github.com:alphagov/govuk-load-testing.git'
+#
+# [*commit*]
+#  The version of the repository where GOV.UK defines its load tests
+#  Default = 'master'
+#
+#
+#
+class govuk::node::s_gatling (
+  $root_dir = '/home/gatling/gatling-software/results',
+  $repo     = 'https://github.com/alphagov/govuk-load-testing.git',
+  $commit   = 'master',
+) inherits govuk::node::s_base {
+
+  # User used to run Gatling
+  govuk_user { 'gatling':
+    fullname => 'GOV.UK Gatling',
+    email    => 'webops@digital.cabinet-office.gov.uk',
+    groups   => ['admin', 'deploy', 'adm', 'www-data' ],
+  }
+
+  # repository where GOV.UK stores its load tests
+  vcsrepo { '/home/gatling/govuk-load-testing':
+    ensure   => present,
+    provider => git,
+    source   => $repo,
+    revision => $commit,
+    force    => true,
+  }
+
+  # install the JRE and JDK on the Gatling instance
+  include ::govuk_java::openjdk8::jre
+  include ::govuk_java::openjdk8::jdk
+  class { 'govuk_java::set_defaults':
+    jdk => 'openjdk8',
+    jre => 'openjdk8',
+  }
+
+  exec { 'get-gatling':
```

Comment:
Can we package this instead using fpm? It'd be better to have a package pulled from our repo than rely on third party sites on each puppet run.

Reply:
yeah, I can